frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-overview/experiment-overview.component.ts

```diff
@@ -92,64 +91,92 @@ export class ExperimentOverviewComponent implements OnInit, OnDestroy {
           this.consistencyRules = [{ value: CONSISTENCY_RULE.INDIVIDUAL }, { value: CONSISTENCY_RULE.EXPERIMENT }];
           break;
         case ASSIGNMENT_UNIT.GROUP:
-          this.overviewForm.get('groupType').enable();
-          this.overviewForm.get('groupType').setValidators(Validators.required);
-          this.consistencyRules = [
-            { value: CONSISTENCY_RULE.INDIVIDUAL },
-            { value: CONSISTENCY_RULE.GROUP },
-            { value: CONSISTENCY_RULE.EXPERIMENT }
-          ];
+          if ( this.overviewForm.get('context').value.length ) {
```

Comment:
@RidhamShah Can you please fix the spacing issue here inside the if condition?

Reply:
code updated